api/networkserver.proto

```diff
@@ -30,6 +33,13 @@ message GenerateDevAddrResponse {
   bytes dev_addr = 1 [(gogoproto.customtype) = "go.thethings.network/lorawan-stack/v3/pkg/types.DevAddr"];
 }
 
+// Request of GetDefaultMACSettings.
+message GetDefaultMACSettingsRequest {
+  string frequency_plan_id = 1 [(gogoproto.customname) = "FrequencyPlanID", (validate.rules).string.max_len = 64];
+  PHYVersion lorawan_phy_version = 2 [(validate.rules).enum.defined_only = true];
+  google.protobuf.FieldMask field_mask = 3;
```

Comment:
I don't know if the field mask really adds that much here. Maybe just leave it out, we can always add it in the future if we say that the default behavior is to return everything.